### PR TITLE
Feature/auction section

### PR DESCRIPTION
**Pull request by W3stside**
added AuctionContainer container/component + story

### CODE REVIEW

File: stories/AuctionContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+import { TokenCode } from 'types'
+
+const Provider = makeProviderDecorator(storeInit())
+
+storiesOf('AuctionContainer', module)
+  .addDecorator(StoryRouter())
+  .addDecorator(Provider)
+  .addWithJSX('PAGE 2', () =>
+    <AuctionContainer auctionDataScreen="amount">
+      <AuctionHeader
```
Comment (Velenir) [inline stories/AuctionContainer.tsx:29]: Any reason not to
```
<AuctionHeader backTo="/" >
   Token Auction ETH/GNO
< /AuctionHeader>
```
?

Comment (W3stside) [reply stories/AuctionContainer.tsx:29]: didn't like that style when writing it. No other pref either way though

Comment (Velenir) [reply stories/AuctionContainer.tsx:29]: good then

**Review by Velenir — APPROVED** [review]

**Comment by W3stside** [comment]
@Velenir let me know what you think of commit https://github.com/gnosis/dutch-exchange/pull/31/commits/6b58805e3bf7e825b868d02de55aab379a9e0b30

**Pull request merged**